plugins/processors/gpuattributes/internal/awsneuron_metric_modifier.go

```diff
@@ -365,3 +318,24 @@ func resetStaleDatapoints(originalMetric pmetric.Metric) {
 		}
 	}
 }
+
+func (md *AwsNeuronMetricModifier) IsProcessedNeuronMetric(name string) bool {
```

Comment:
I dont think its the correct place for this function, it has no use case for metric modifier

Reply:
created a new class for it